release-schema.json

```diff
@@ -34,7 +34,8 @@
             "title":"Paid by",
             "description":"Is this a user charge (paid by businesses or citizens using the facilities provided by the contract), or a charge paid by the government?",
             "type":["string","null"],
-            "enum":["government","user"]
```

Comment:
Just confirming that it was not the original intent to have a closed codelist?

Reply:
A good question. I don't think there is a strong intent either way. 

The PPP framework just specifies government vs user changes - and I'm assuming this is the basis for certain analysis to be done.

As the only use-case for this is currently the PPP framework, I've switched back to closed, as it is I think easier to relax to an open codelist in future, than to go the other way.